ui/ui.go

```diff
@@ -225,6 +225,12 @@ type dest struct {
 func (o *Object) init() {
 	var p interface{}
 	switch o.Kind {
+	case "bandLimitedSaw":
```

Comment:
Yeah I wasn't sure if that's what you want or not, but they definitely sound a lot cleaner, though the saw is still a bit thin and buzzy for my taste, but that's probably something to tackle later. I'll go ahead and remove the existing square and then rename these then.